### DISCUSSION TITLE
🎉Who is using Ant Design Blazor in production? 👩🏻‍💻

### DISCUSSION THREAD

**Discussion by ElderJames** [ant-design-blazor/ant-design-blazor#1693] [Show and tell]
We appreciate your support if you or your organization is using Ant Design Blazor. You are welcome to leave replies about your product and organization here, which could became the confidence of maintiners, communication and undecided watchers.

Recommended reply format:

```
- Product：
- Company or Organization: (if any)
- Link: (if any)
- Screenshot: (if any)
```

**Comment by StardustDL** [comment]
I'm writing a blog system based on Blazor these days, have tried some component libraries, and finally selected Ant Design Blazor.

- Product: [AcBlog](https://github.com/acblog/acblog)
- Online demo: 
  - [GitHub](https://acblog.github.io/)
  - [Gitee](https://acblog.gitee.io/)

Thank you for maintaining this project. Looking forward to the final release.

![](https://github.com/StardustDL/own-staticfile-hosting/raw/master/acblog/images/preview-home.png)
![](https://github.com/StardustDL/own-staticfile-hosting/raw/master/acblog/images/preview-post.png)
![](https://github.com/StardustDL/own-staticfile-hosting/raw/master/acblog/images/preview-archive.png)
![](https://github.com/StardustDL/own-staticfile-hosting/raw/master/acblog/images/preview-search.png)

**Reply by TarasOnyshko** [reply]
@StardustDL Nice website. Were you able to override the default primary color (blue) to another color? Just need to find a way to do that for my Blazor app, thanks.

**Comment by Seanxwy** [comment]
Startup Company , e-commerce project. use ant design blazor. blazor server side coding
Product: mall manage project 
Screenshot: coming soon
Expected on-line time: 2020/08

**Reply by ElderJames** [reply]
Hi @Seanxwy , If you could provide screenshots, we would be appreciate.

**Comment by ske66** [comment]
Product: Aviagen Data Manager
Company: Aviagen Ltd
Link: unfortunately not available past the VPN
![DataManager](https://user-images.githubusercontent.com/26600892/87009357-f6fcda80-c1bc-11ea-87ad-c27720d9cef2.png)

**Comment by aaasoft** [comment]
Product：Data collect gateway
Company or Organization: 弈力科技
Link: http://www.willservice.net/
Screenshot: 
![QQ截图20200710111917](https://user-images.githubusercontent.com/1552027/87112758-3f8cc480-c29f-11ea-9089-c046019eaa7a.png)
![QQ截图20200710112103](https://user-images.githubusercontent.com/1552027/87112843-782c9e00-c29f-11ea-9e4a-13759427f27d.png)

**Comment by 1002527441** [comment]
Product: EM Services
Company: Wurth Electronic (shenzhen) Ltd (www.we-online.com)
Link:  using for internal.

![EMServices](https://user-images.githubusercontent.com/53967483/87378662-dff52880-c5c0-11ea-90c0-4960337d8f88.png)

![Permission](https://user-images.githubusercontent.com/53967483/87378823-53973580-c5c1-11ea-9e4b-a23d3c41bf67.png)
![Samplemanage](https://user-images.githubusercontent.com/53967483/87378825-5560f900-c5c1-11ea-8af8-1f0026b5b1b2.png)
![EquipmentQueyr](https://user-images.githubusercontent.com/53967483/87378849-6447ab80-c5c1-11ea-97ad-443799586577.png)

dashboard ( we used chart component of AntDesign-chart-blazor )

![image](https://user-images.githubusercontent.com/53967483/90207655-6ba8e180-de19-11ea-9215-41a32de79188.png)
![image](https://user-images.githubusercontent.com/53967483/90207689-80857500-de19-11ea-837d-c8c225d532eb.png)

**Comment by imhmao** [comment]
Product：SCM

**Reply by ElderJames** [reply]
Hi @imhmao , If you could provide screenshots, we would be appreciate.

**Comment by wcoolly** [comment]
- Product：SoftPortal
- Company or Organization: Qingdao Corill Electronic Ltd
- Link: http://www.corill.com

**Reply by ElderJames** [reply]
Hi @wcoolly , If you could provide screenshots, we would be appreciate.

**Comment by pearl2201** [comment]
- Product: Book Store Dashboard
- Company: Kyx Studio JSC

![Annotation 2020-07-27 104128](https://user-images.githubusercontent.com/5409440/88501635-3a957800-cff6-11ea-9d15-e331f5ba9c52.png)
![Annotation 2020-07-27 104301](https://user-images.githubusercontent.com/5409440/88501650-42edb300-cff6-11ea-8c29-29377044d98a.png)
![Annotation 2020-07-27 104153](https://user-images.githubusercontent.com/5409440/88501654-4719d080-cff6-11ea-9237-106b003dc05f.png)
![Annotation 2020-07-27 104214](https://user-images.githubusercontent.com/5409440/88501720-72042480-cff6-11ea-8de2-b7b70a141fc1.png)

**Comment by itmaybetrue** [comment]
Product：AI-WMS
Company: amtxts Ltd
![微信截图_20200930172254](https://user-images.githubusercontent.com/33754438/94667636-a4f1ce00-0341-11eb-8be4-9ffbcbc66ec4.png)

**Comment by TimChen44** [comment]
- Product：组织绩效数据管理平台
- Company：Chint
- Link: http://www.chint.net/zh/index.php/about/index.html
![screencapture-10-10-14-70-2023-AchievementMgr-AchievementSearch-2020-09-30-14_06_13](https://user-images.githubusercontent.com/7581981/95337122-20282680-08e4-11eb-9a99-3d1e66486b54.png)
![screencapture-10-10-14-70-2023-IndexMgr-IndexSearch-2020-09-30-14_13_01](https://user-images.githubusercontent.com/7581981/95337132-21595380-08e4-11eb-88ef-293f74852d71.png)

**Comment by justmine66** [comment]
Thanks
Product: CMS
Company: 际客国际
Link: using for internal.
![image](https://user-images.githubusercontent.com/23122801/100435290-b274cb80-30d8-11eb-86ba-b2913c6644bb.png)
![image](https://user-images.githubusercontent.com/23122801/100435406-da642f00-30d8-11eb-84e1-fc21129947a5.png)
![image](https://user-images.githubusercontent.com/23122801/100435478-f36ce000-30d8-11eb-9c4c-e63bee2f70dc.png)

**Comment by CSharpXu** [comment]
既要用ASP.NET Core Blazor，又要符合UI设计要求，最终选择了Ant Design Blazor，谢谢James，谢谢团队！当我们遇到问题及时作出反馈。

**Reply by ElderJames** [reply]
Hi @CSharpXu , If you could provide screenshots, we would be appreciate.

**Comment by jingxi0109** [comment]
Product: Review System
Company: Ai Win Sales
Link: aiwinsales.com
![屏幕截图 2020-11-27 183018](https://user-images.githubusercontent.com/18211444/100439539-a3911780-30de-11eb-8eb7-846feb41a168.png)

**Comment by fan0217** [comment]
- Product：Financial Management System
- Company or Organization: XFly
- Link: 
![Blazor](https://user-images.githubusercontent.com/13997023/103061958-bb798f80-45e7-11eb-9957-02459184a496.png)

**Comment by guanshuqing** [comment]
产品：冠新内控系统
公司：冠新软件
链接：无（公司内部系统不对外）

**Reply by ElderJames** [reply]
Hi @guanshuqing , If you could provide screenshots, we would be appreciate.

**Comment by akinix** [comment]
@fan0217 你的UI比较简洁 是自定义的么?

**Comment by richerp** [comment]
产品：企业Erp系统
公司：RichErp
链接：www.richerp.com

**Reply by ElderJames** [reply]
Hi @richerp , If you could provide screenshots, we would be appreciate.

**Comment by anddrzejb** [comment]
Product: Fare Montior system
Company: Intercity Cyprus
No link, internal app
Shameless ripoff of Pro template 😏

**Reply by ElderJames** [reply]
Are there any screenshot? ☺️Looking forward that.

**Comment by bryanrcarlson** [comment]
- Product：[Midden: Research Data Catalog](https://github.com/CafIncubator/Midden)
- Online example: [Cook Agronomy Farm Metadata](https://meta.cafltar.org)
- Screenshot: 
  - ![Insights dashboard](https://github.com/CafIncubator/Midden/raw/main/media/ss-insights.jpg)
  - ![Dataset details](https://github.com/CafIncubator/Midden/raw/main/media/ss-dataset-view.jpg)

**Comment by newbe36524** [comment]
- Product: Amazing-Favorites is a browser extension to help you to manage your bookmarks efficiently.
- Link:  https://github.com/newbe36524/Amazing-Favorites

You can install this extension from the stores below:

Chrome extension store: <https://chrome.google.com/webstore/detail/amazing-favorites/podhpclhgkdeiechkdceginfehfanhcb>
Microsoft extension store: <https://microsoftedge.microsoft.com/addons/detail/amazing-favorites/bknjgbpkaloajcphccpcnahegfglfiei>

- Screenshot: 

![open_manager](https://user-images.githubusercontent.com/7685462/116783869-b8c20600-aac3-11eb-88fb-a2b4aaf32304.gif)
![add_bookmark](https://user-images.githubusercontent.com/7685462/116783871-b9f33300-aac3-11eb-896f-d1c05893c671.gif)

**Comment by Alexandre789010** [comment]
Hi, I use AntBlazor in production. (Internal software)
Cannot share the repo unfortunately - but here are some pictures:

![image](https://user-images.githubusercontent.com/60594505/123912511-78e2a680-d9c0-11eb-892d-b71757c4004c.png)

![image](https://user-images.githubusercontent.com/60594505/123912600-9879cf00-d9c0-11eb-9d17-1c762f97ff58.png)

![image](https://user-images.githubusercontent.com/60594505/123912666-ad566280-d9c0-11eb-85cd-1125b0a0303e.png)

![image](https://user-images.githubusercontent.com/60594505/123912980-06be9180-d9c1-11eb-9cd9-c872ed5df906.png)

![image](https://user-images.githubusercontent.com/60594505/123913133-353c6c80-d9c1-11eb-9701-746e1c9ff947.png)

![image](https://user-images.githubusercontent.com/60594505/123912808-d840b680-d9c0-11eb-9d7a-4f511f1da90c.png)


Thank you to all the contributors. Amazing job! 👍

**Reply by akhanalcs** [reply]
Wow, looks pretty amazing! Did you use nested tables in it? 
I'd love to see an example.
I couldn't find the documentation for nested tables in the website though.

**Reply by ElderJames** [reply]
@affableashish I am adding the nested table demo for documentation, see https://github.com/ant-design-blazor/ant-design-blazor/pull/1884

**Reply by TarasOnyshko** [reply]
@Alexandre789010 Cool! Were you able to override the default primary color(blue) ?

**Reply by akhanalcs** [reply]
> @affableashish I am adding the nested table demo for documentation, see #1884

Thank you so much. The nested tables look so neat!

**Comment by JiaoLongYang** [comment]
- Product：运输企业安全生产信息管理平台
- Screenshot: 
![image](https://user-images.githubusercontent.com/20771689/123913297-9d368700-d9b0-11eb-88fa-12abff6fb8b4.png)
![image](https://user-images.githubusercontent.com/20771689/123913484-da027e00-d9b0-11eb-8073-b0f841b7f229.png)
![image](https://user-images.githubusercontent.com/20771689/123914092-99efcb00-d9b1-11eb-8d90-ec02fb2f5a49.png)
![image](https://user-images.githubusercontent.com/20771689/123916383-2bf8d300-d9b4-11eb-92cf-a4ba74f9f1e2.png)

**Reply by JustGentle** [reply]
怎么做到ScorllX ScorllY 同时自适应宽度高度的！很不错！

**Comment by BeiYinZhiNian** [comment]
- Product：Caviar   (It's still under development)
- Link: https://gitee.com/Cherryblossoms/caviar
           https://github.com/BeiYinZhiNian/Caviar
- Screenshot: 
![image](https://user-images.githubusercontent.com/25119211/123916583-66627000-d9b4-11eb-97f8-0f10c41be10a.png)
![image](https://user-images.githubusercontent.com/25119211/123916970-d7098c80-d9b4-11eb-96e9-e34cb05a8080.png)
![image](https://user-images.githubusercontent.com/25119211/123917049-edafe380-d9b4-11eb-81b8-532d813568c8.png)
![image](https://user-images.githubusercontent.com/25119211/123917210-1df78200-d9b5-11eb-83c7-e6f53dca2dab.png)
![image](https://user-images.githubusercontent.com/25119211/123917233-23ed6300-d9b5-11eb-8327-f18bf2c6e2c3.png)
![image](https://user-images.githubusercontent.com/25119211/123917267-2e0f6180-d9b5-11eb-9a89-7d66d3a009d0.png)

**Comment by LingDian2019** [comment]
Product：WCS
Company or Organization: 碧桂园
Link: WCS项目（内网）
Screenshot:
![image](https://user-images.githubusercontent.com/49890738/128108642-444fdfc9-e685-46cc-8f26-222de261e258.png)
![image](https://user-images.githubusercontent.com/49890738/128108693-a0cd581e-5dc8-4baa-8fa8-3e3abd3d1210.png)
![image](https://user-images.githubusercontent.com/49890738/128108778-b376221c-cf5e-4018-8689-e0c8d711f4b2.png)

**Comment by huhangfei** [comment]
Product：Gardener
Link: https://gitee.com/hgflydream/Gardener
Screenshot:
![101651_4fed42b7_302533](https://user-images.githubusercontent.com/7147335/130720944-d19b0ef2-3ce7-4697-8074-a1d5d3d2174a.png)

![101709_a9342323_302533](https://user-images.githubusercontent.com/7147335/130720973-83759bc3-530f-4cc9-acd9-422541b36232.png)
![101720_47067234_302533](https://user-images.githubusercontent.com/7147335/130720982-53c944ef-9b26-4482-9978-a64b7346bdae.png)
![101731_7dd825d7_302533](https://user-images.githubusercontent.com/7147335/130720998-78ebffcd-d26a-4301-b3ac-f65610a41ba9.png)
![101742_918c9d0a_302533](https://user-images.githubusercontent.com/7147335/130721010-3e97b6b3-e8f8-44ab-a2e6-18fe08312b27.png)
![101926_87c9983b_302533](https://user-images.githubusercontent.com/7147335/130721021-5834019b-fe29-4666-86cb-32ff7d9f1d2e.png)
![101940_8e831f99_302533](https://user-images.githubusercontent.com/7147335/130721033-83d2866d-1f17-4aa2-94d0-99ab0e222f15.png)
![101951_6a928009_302533](https://user-images.githubusercontent.com/7147335/130721055-e13b41c5-96f9-4498-852b-1e7622246006.png)

**Comment by gosszhang** [comment]
产品：基础设施门户站点
公司：校宝在线
截图：
![image](https://user-images.githubusercontent.com/14506118/130904487-23cb8e25-6d1d-4642-92c0-860ebd6975b6.png)

**Comment by Hona** [comment]
# **UpBlazor**
- Link: https://upblazor.com

Screenshots:

![image](https://user-images.githubusercontent.com/10430890/146133783-d46bc11a-b5a3-4190-b17a-9256ecc8da22.png)
![image](https://user-images.githubusercontent.com/10430890/146133894-c4770784-1e14-4019-bc44-96f564ab6c28.png)
![image](https://user-images.githubusercontent.com/10430890/146133838-f1f41bf1-031e-44f9-82d5-fd1aec7c5f83.png)
![image](https://user-images.githubusercontent.com/10430890/146133936-5407cd24-cff1-4e6b-be2e-5aa57875286d.png)
![image](https://user-images.githubusercontent.com/10430890/146133960-99c5cfa2-b028-4be9-b498-33d4bf46c12d.png)
![image](https://user-images.githubusercontent.com/10430890/146133975-d3c74741-4058-46c9-942f-648075c46c0f.png)
![image](https://user-images.githubusercontent.com/10430890/146134004-cec73284-a586-43f0-b2fc-2c1b329b5445.png)
![image](https://user-images.githubusercontent.com/10430890/146134047-16eafddf-69fe-4c7e-9e6a-b14f80840e1d.png)
![image](https://user-images.githubusercontent.com/10430890/146134064-2a6fea50-414f-405c-8f73-7ed1a2be511a.png)
![image](https://user-images.githubusercontent.com/10430890/146134104-a63292a6-9653-43a8-bd09-8054f6242d92.png)
![image](https://user-images.githubusercontent.com/10430890/146134135-32a19869-aeac-499f-80bf-be8fac31ad91.png)
![image](https://user-images.githubusercontent.com/10430890/146134151-63c2c349-c919-470f-bec6-d37624f1e901.png)
![image](https://user-images.githubusercontent.com/10430890/146134268-d30762c1-4c21-4fd4-9912-5138f767c81e.png)

**Comment by xisimao** [comment]
魔兽战斗分析系统
自己没事做着玩
代码地址：https://gitee.com/xisimao/wra
![微信截图_20211230173018](https://user-images.githubusercontent.com/16586355/147739234-a25e6a82-ee85-4294-a568-9c2a5f088fe8.png)
![微信截图_20211230172718](https://user-images.githubusercontent.com/16586355/147739049-18d0b11f-45cf-48ff-a7f2-ae5c49b13a0f.png)
![微信截图_20211230172722](https://user-images.githubusercontent.com/16586355/147739050-f7a869d4-27e0-449e-a890-7b0228237f77.png)

**Comment by bofwind** [comment]
A production demonstrate and maintenance UI:
![res-1](https://user-images.githubusercontent.com/48547129/161384343-98a43a1a-96c6-460b-816e-5243780f1d22.PNG)
![res-3](https://user-images.githubusercontent.com/48547129/161384607-1d33e442-8fdb-4ec7-ada6-ce2d88a163bf.PNG)

![res-2](https://user-images.githubusercontent.com/48547129/161384345-aa36e87d-e28f-4b9b-b37f-c75924d403ba.PNG)

**Reply by fagro-vizcaino** [reply]
Do you have any URL for this? 
Is this Wasm or server?

**Reply by bofwind** [reply]
> Do you have any URL for this? Is this Wasm or server?

first two snapshots are for public site, currently your can access it from ["here"](https://respackageing-eastasia.azurewebsites.net/ProductQuery/00000000-0000-0000-0000-000000000001/1), it still under developing, not finish yet. so only product query feature is available. (it was developed as a wasm app, but run as hybrid mode: display server sider render first, after wasm assemblies were fully loaded, auto switch to wasm mode)

the 3rd snapshot is for backend and run at server side. it's not for public access.

**Reply by fagro-vizcaino** [reply]
Thanks!!
It seems very responsive with that hybrid mode, do you have any docs about that hybrid mode that you used?

**Reply by bofwind** [reply]
> Thanks!! It seems very responsive with that hybrid mode, do you have any docs about that hybrid mode that you used?

i was base on this guy's work: [https://blog.csdn.net/qq_32174441/article/details/122350559](https://blog.csdn.net/qq_32174441/article/details/122350559)

this article is in chinese, basicly, it need you create your blazor application in wasm mode first, then create a server side application refer the wasm one, and modify the _Host.cshtml, base on a querystring from URL to decide use server side mode or wasm, for wasm mode, embed an iframe for server side mode, and made the iframe full of the whole page, when wasm was loaded, invoke js to hide the iframe finished the switch. of coz, there're many details i didn't mentioned here, you can find them in the article.

**Comment by zizaixiuxing** [comment]
- Product：A tutoring exercise system written for my students
-  by a middle school teacher

-
![a](https://user-images.githubusercontent.com/53082806/163812467-8f14e474-9fb8-4c6b-a23f-5b00a5bcaa8c.png)
![b](https://user-images.githubusercontent.com/53082806/163812476-68c1e7e2-ba2f-472c-92b6-6a84f65f2aa5.png)

**Comment by chunliu** [comment]
I am using antd in my project: https://www.azuredesign.app/ 

![image](https://user-images.githubusercontent.com/2128551/169830861-e65f4f9d-a033-4f85-beb1-adc67e31fb34.png)

**Reply by ElderJames** [reply]
It's very cool! I will post it in twitter.

**Reply by chunliu** [reply]
Thanks @ElderJames

**Comment by mick-xu** [comment]
- Product：Rapid Site
- Company or Organization: 济南颖华信息技术有限公司
- Link: https://www.yinghuasoft.com https://www.yinghuainfo.com
- Screenshot:

![image](https://user-images.githubusercontent.com/3981012/217730687-316f09c4-3096-4969-8126-c549be9a5b5b.png)
![image](https://user-images.githubusercontent.com/3981012/217730879-9a9b65e5-74e1-45e7-af31-547c467f3891.png)

**Comment by known** [comment]
- Product：Known快速开发框架
- Company or Organization: 普漫科技
- Link: [http://known.pumantech.com/](http://known.pumantech.com/)
- Screenshot: 
![image](https://github.com/ant-design-blazor/ant-design-blazor/assets/4126663/fd69fd9a-d56f-4c40-a265-57c221b5cafd)
![image](https://github.com/ant-design-blazor/ant-design-blazor/assets/4126663/3ec011cc-df4d-483a-bc68-c452cdee9c5d)
![image](https://github.com/ant-design-blazor/ant-design-blazor/assets/4126663/3712b6b3-9964-4d21-a78e-520cc8265448)
![image](https://github.com/ant-design-blazor/ant-design-blazor/assets/4126663/5e468067-bc39-4614-9dff-aa0c9e915d00)
![image](https://github.com/ant-design-blazor/ant-design-blazor/assets/4126663/d3e0fb86-2459-4e89-aaae-9022fe4a5f06)

**Comment by bxjg1987** [comment]
- Product：bxjg
- Company or Organization: bxjg
- Link:
   [code cource](https://gitee.com/bxjg1987_admin/abp#%E7%AE%80%E4%BB%8B)
   [demo](http://zlj.cqyuzuji.com:19911/)
- Screenshot:
![image](https://github.com/ant-design-blazor/ant-design-blazor/assets/39435771/8a4c2086-a4b2-4aaf-ac93-6aea8192c77e)

**Comment by jstafford5380** [comment]
Hi, we are a large retailer in the U.S. ($6B+, 15k employees) and  we are using AntBlazor in production for an internal tool that is similar to Backstage by Spotify. I cannot share more than that unfortunately :)
